docs/external-oasis-repos.md

```diff
@@ -67,34 +79,33 @@ Requests to Remove an External Repository
 -----------------------------------------
 
 1.  After validating that the ticket submitter is authorized by the VO's OASIS manager, delete the registered value
-    for %RED%example.opensciencegrid.org%ENDCOLOR% in topology for the VO in OASIS Repo URLs.
+    for <EXAMPLE.OPENSCIENCEGRID.ORG> in topology for the VO in OASIS Repo URLs.
     Verify that it is removed by running the following on any oasis machine
     to make sure it is missing from the list:
 
         :::console
-        print_osg_repos|grep %RED%example.opensciencegrid.org%ENDCOLOR%
+        print_osg_repos|grep <EXAMPLE.OPENSCIENCEGRID.ORG>
 
-1.  Add the FNAL and BNL Stratum-1 operators to the ticket and ask them to remove the repository. Wait for the
-    Stratum-1 operators to finish before proceeding.
-1.  Run the following commands on `oasis-replica-itb` and `oasis-replica`:
+1.  Add the BNL Stratum-1 operator to the ticket and ask him to remove the repository. Wait for
+    him to finish before proceeding.
+1.  Run the following command on `oasis`, `oasis-itb`, `oasis-replica` and `oasis-replica-itb`:
```

Comment:
same here, somehow make it clear what's the real hostname?

Reply:
In think in this case the short names are clear enough.